test/integration/blocks-raw-handling.test.js

```diff
@@ -57,6 +56,28 @@ describe( 'Blocks raw handling', () => {
 			},
 			save: () => null,
 		} );
+
+		registerBlockType( 'test/iframe-handler', {
```

Comment:
I see that it's been done previously, but why are we adding this block type for every test? Shouldn't we isolate it to the test?